DOCS.md

```diff
@@ -412,6 +413,18 @@ login({appState: JSON.parse(fs.readFileSync('appstate.json', 'utf8'))}, (err, ap
 
 ---------------------------------------
 
+<a name="forwardAttachment"></a>
+### api.forwardAttachment(attachmentID, userOrUsers[, callback])
+
+Takes a userID or an array of userIDs and forward the corresponding attachment.
+
+__Arguments__
+* `attachmentID`: A attachment ID (usualy `attribute.ID` property)
```

Comment:
Could you rephrase this to be:

The `ID` field in the attachment object. Not all attachment have `ID`s: recorded audio and arbitrary files don't for example.